src/main/resources/db/migration/V1__imrt_ddl.sql

```diff
@@ -0,0 +1,70 @@
+/* Primary table containing all types of items and the json data from IAT */
+CREATE TABLE item (
+  key             SERIAL PRIMARY KEY, -- Primary key for items, auto-incrementing (change serial to AUTO_INCREMENT for mySQL)
+  id              VARCHAR(9)  NOT NULL, -- Item ID as specified by IAT
+  subject         VARCHAR(20) NOT NULL,
+  grade           VARCHAR     NOT NULL,
+  workflow_status VARCHAR     NOT NULL,
+  item_type       VARCHAR     NOT NULL,
+  dok             VARCHAR     NOT NULL,
+  item_created_by VARCHAR     NOT NULL,
+  item_created_at VARCHAR     NOT NULL,
+  item_json       JSON        NOT NULL,
+  created_at      TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_at      TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_by      VARCHAR     NOT NULL
+);
+
+CREATE TABLE standard_id (
+  item_key             INT         NOT NULL,
+  type                 INT         NOT NULL, -- Primary = 1, Secondary = 2, etc
+  claim                VARCHAR     NOT NULL,
+  content_domain       VARCHAR     NOT NULL,
+  target               VARCHAR     NOT NULL,
+  emphasis             VARCHAR     NOT NULL,
+  common_core_standard VARCHAR     NOT NULL,
+  created_at           TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_at           TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_by           VARCHAR     NOT NULL,
+  FOREIGN KEY (item_key) REFERENCES item (key),
+  PRIMARY KEY (item_key, type)
+);
+
+/* Git Information associated with each item */
+CREATE TABLE item_git (
+  item_key            INT         NOT NULL,
+  project_id          INT PRIMARY KEY, -- Unique identifier from GIT
+  project_path        VARCHAR     NOT NULL, -- Full path including namespace (group)
+  current_commit_hash VARCHAR     NOT NULL,
+  created_at          TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_at          TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_by          VARCHAR     NOT NULL,
+  FOREIGN KEY (item_key) REFERENCES item (key),
+  UNIQUE (item_key, project_id)
+);
+
+/* Historical item data */
+CREATE TABLE item_log (
+  item_key    INT         NOT NULL,
+  commit_hash VARCHAR     NOT NULL,
+  commit_date TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  created_at  TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_at  TIMESTAMPTZ NOT NULL DEFAULT current_timestamp,
+  updated_by  VARCHAR     NOT NULL,
+  FOREIGN KEY (item_key) REFERENCES item (key),
+  PRIMARY KEY (item_key, commit_hash)
+);
+
+/* Link Assessment items and stimulii */
+CREATE TABLE stim_link (
+  item_key      INT         NOT NULL,
+  item_id       INT         NOT NULL,
```

Comment:
I'm thinking the IAT ids would be helpful in this table, as well as our internal key values. If you agree I can add an index on them as well.

Reply:
Can you give me a scenario that we're currently supporting that would dictate needing this value?  An example would help me understanding your thinking.

Reply:
It would allow us to avoid a join if we want to count the number of items associated with a given stimulus ID. Or list the items associated with a stim.